pages/instructorQuestionManualGrading/instructorQuestionManualGrading.ejs

```diff
@@ -22,6 +22,32 @@
     </div>
     <div id="content" class="container">
       <div class="row">
+
+        <% if (typeof submission != null) { %>
+          <% if (submission.score == null && submission.manual_grading_user != null) { %>
+          <div class="col-lg-12 col-sm-12">
```

Comment:
Is this `<div>` needed? What's it doing?

Reply:
It defines a grid as full across for the badge.

![image](https://user-images.githubusercontent.com/7872295/108889333-1d8f4a80-75c1-11eb-8897-d6937bab3d08.png)

Reply:
So if we're not doing the "You are currently grading..." banner, then this can go.

Reply:
Removed.